wireless_modules/test.py

```diff
@@ -0,0 +1,57 @@
+import paho.mqtt.client as mqtt
+import time
+import argparse
+# Note: Change to local host at the end
+
+parser = argparse.ArgumentParser(
+    description='MQTT wireless module test script with dummy payloads',
+    add_help=True)
+parser.add_argument(
+    '--module', action='store', type=int,
+    default=2, help="""The module number to determine the publish and subscription topics. 
+    If nothing is selected it will send on module number 2 topics.""")
+parser.add_argument(
+    '--host', action='store', type=str, default="localhost",
+    help="""Address of the MQTT broker. If nothing is selected it will
+    default to 5.196.95.208 - a test server by mosquito.""")
```

Comment:
No need to mention what it will default to. `argparse` will show you when you do a `--help` anyway. The address here is incorrect anyway.